wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,137 @@
+# Difficulty Naming
+
+This wiki article serves as a substitute and an update to [this topic](https://osu.ppy.sh/community/forums/topics/178700). It contains the ways in which difficulties in each respective game mode are commonly named, as well as various other alternatives that are also allowed.
+
+## Default	
+
+This subsection shows the default scaling difficulty names for each game mode, which are the most commonly used.
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+ 
+- ![Easy](/wiki/shared/diff/easy-s.png "Easy") Easy 
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hard](/wiki/shared/diff/hard-s.png "Hard") Hard
+- ![Insane](/wiki/shared/diff/insane-s.png "Insane") Insane
+- ![Extra](/wiki/shared/diff/expert-s.png "Extra") Extra
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Kantan](/wiki/shared/diff/easy-t.png "Kantan") Kantan
+- ![Futsuu](/wiki/shared/diff/normal-t.png "Futsuu") Futsuu
+- ![Muzukashii](/wiki/shared/diff/hard-t.png "Muzukashii") Muzukashii
+- ![Oni](/wiki/shared/diff/insane-t.png "Oni") Oni
+- ![Inner Oni / Ura Oni](/wiki/shared/diff/expert-t.png "Inner Oni / Ura Oni") Inner Oni / Ura Oni
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
+
+- ![Cup](/wiki/shared/diff/easy-c.png "Cup") Cup 
+- ![Salad](/wiki/shared/diff/normal-c.png "Salad") Salad
+- ![Platter](/wiki/shared/diff/hard-c.png "Platter") Platter
+- ![Rain](/wiki/shared/diff/insane-c.png "Rain") Rain
+- ![Overdose](/wiki/shared/diff/expert-c.png "Overdose") Overdose
+
+### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
+
+- ![EZ](/wiki/shared/diff/easy-m.png "EZ") EZ 
+- ![NM](/wiki/shared/diff/normal-m.png "NM") NM
+- ![HD](/wiki/shared/diff/hard-m.png "HD") HD
+- ![MX](/wiki/shared/diff/insane-m.png "NX") MX
+- ![SC](/wiki/shared/diff/expert-m.png "SC") SC
+
+## Alternative
```

Comment:
Are these actual variants or inserts? For example, you say Advanced is between Normal and Hard, so could you have a set that goes ENAHIX? For me, a variant suggests an alternative difficulty naming **set** rather than individual diff names, so I would choose my words carefully, since there appears to be some inconsistency here (especially since you are drawing comparisons to standard ENHIX)

For Taiko and Catch it is pretty clear they are additional inserts, but for osu! I'm not so sure that's what's being conveyed here...

Reply:
but those are individual names, not sets at all... where do you see the set there? every single bulletpoint in Alternative means exactly that: alternative ways of naming a difficulty that is not exactly fulfilling neither the criteria of the above nor the below respective tier of difficulty. Those are not sets, the ones that are sets (spreads, more correctly) are the ones that belong to other rhythm games mostly